docs/user/rstudio/ide/guide/productivity/text-editor.qmd

```diff
@@ -54,22 +54,34 @@ You can comment and uncomment entire selections of code using the **Code -\> Com
 
 ![](images/multicursor-lines.png){fig-alt="Screenshot of multiple cursors across 3 lines of code"}
 
-You can also use the "All" button on the **Find/Replace** toolbar to create cursors at each instance of a word.
+Use the "All" button on the **Find/Replace** toolbar to create cursors at each instance of a word.
 
 ![](images/multicursor-find-replace.png){fig-alt="Screenshot of 3 cursors, one at each instance of the searched term in the current R script."}
 
+## Move lines up/down
+
+Text or code in source files can be moved up or down by the <kbd>Alt+Up/Down</kbd> (<kbd>Option+Up/Down</kbd> on Mac). This applies to individual lines of code where the cursor is, as well as to multiline selections.
+
+![](images/move-lines-up-down.gif){fig-alt="A screencast of an example line of text being moved up with the move lines up/down shortcut"}
+
+## Copy lines up/down
+
+Text or code in source files can be moved up or down with the <kbd>Shift+Alt+Up/Down</kbd> (<kbd>Cmd+Option+Up/Down</kbd> on Mac) shortcut. This applies to individual lines of code where the cursor is, as well as to multiline selections.
+
```

Comment:
This text is identical to the **Move lines up/down** section above. Maybe say something like "Text or code in source files can be copied and moved down..." to distinguish it.

Reply:
Oof - thank you!